agent/src/test/java/com/navercorp/pinpoint/plugin/hystrix/HystrixCommand_1_5_x_IT.java

```diff
@@ -0,0 +1,95 @@
+package com.navercorp.pinpoint.plugin.hystrix;
+
+import com.navercorp.pinpoint.bootstrap.plugin.test.Expectations;
+import com.navercorp.pinpoint.bootstrap.plugin.test.ExpectedAnnotation;
+import com.navercorp.pinpoint.bootstrap.plugin.test.PluginTestVerifier;
+import com.navercorp.pinpoint.bootstrap.plugin.test.PluginTestVerifierHolder;
+import com.navercorp.pinpoint.test.plugin.Dependency;
+import com.navercorp.pinpoint.test.plugin.PinpointPluginTestSuite;
+import com.netflix.hystrix.Hystrix;
+import com.netflix.hystrix.HystrixCommand;
+import junit.framework.Assert;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.lang.reflect.Method;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+
+import static com.navercorp.pinpoint.bootstrap.plugin.test.Expectations.annotation;
+
+/**
+ * Created by jack on 7/4/16.
+ */
+@RunWith(PinpointPluginTestSuite.class)
+@Dependency({"com.netflix.hystrix:hystrix-core:1.5.3","com.netflix.hystrix:hystrix-metrics-event-stream:1.1.2"})
```

Comment:
Just an idea - could this be tested using for hystrix-core versions 1.5.3+? Like so:
`com.netflix.hystrix:hystrix-core:[1.5.3,)`
It would be useful to test against future releases as test failure on a new release will let us know that we need to update the plugin.